modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -0,0 +1,515 @@
+= snowflake_streaming
+// tag::single-source[]
+:type: output
+:page-beta: true
+:categories: ["Services"]
+
+component_type_dropdown::[]
+
+NOTE: For improved performance, cost effectiveness and ease of use, Redpanda recommends that you use this output instead of `snowflake_put`.
```

Comment:
I'm not sure this adds value here. The user is already on this page. They aren't looking at `snowflake_put` so it's distracting to be told about it when we aren't recommending it.

Reply:
Updated.